src/features/gdpr.py

```diff
@@ -138,14 +137,7 @@ def _start(self, website_data: WebsiteData) -> dict:
         ]:
             values += func(website_data=website_data)
 
-        flat_values = []
-        for value in values:
-            if isinstance(value, list):
-                flat_values.extend(list(value))
-            else:
-                flat_values.append(value)
-
-        return {VALUES: list(set(flat_values))}
+        return list(set(values))
```

Comment:
Does this flatten the list? The above code looks different in functionality, to me.